.eslintrc.js

```diff
@@ -145,6 +145,7 @@ module.exports = {
           'Badge',
           'HeaderActionButton',
           'Pill',
+          'List.Item',
```

Comment:
Let's keep an eye on this rule to make sure we're not jamming in too many things 🕵️

Reply:
Yes of course, I am subscribed to the issue tracking this in their repo :wink: